src/plugin.js

```diff
@@ -90,7 +90,8 @@ function dispatchEvent(chart, listeners, label) {
   }
 }
 
-function dispatchMoveEvents(chart, listeners, previous, label) {
+// eslint-disable max-params
```

Comment:
This disables `max-params` for all the rest of the file, not only the next line. The correct format is `eslint-disable-next-line  max-params`. However, it doesn't seem needed anymore since the error was from CC so we should just remove it.

Reply:
I would prefer to remove it. What do you think?

Reply:
DONE